### ISSUE TITLE
[BUG] accent dropdown menu not showing for nan-tw

### ISSUE THREAD

**Issue by irvin** [common-voice/common-voice#3708]
**Describe the bug**
The accent menu not showing up for nan-tw. 

We had added the accent list at https://github.com/common-voice/common-voice/pull/3473 but it still not showing up yet. Did I make any mistake that cause this to happen (such as the non-upper case in the locale code?)

**To Reproduce**
1. Go to https://commonvoice.mozilla.org/en/profile/info
2. Click Add languages, select `臺語`
3. Click `How would you describe your accent?` textbox 

**Expected behavior**
A dropdown list containing list items from #3473 should show up

**Screenshots**

_click inputbox of nan-tw not brining up default accent dropdown list_
<img width="611" alt="截圖 2022-05-18 上午3 02 10" src="https://user-images.githubusercontent.com/165936/168890330-374bfea3-9d34-43fc-97ef-8e247a01b3bf.png">

_expected behavior (same list as click at zh-tw)_
<img width="611" alt="截圖 2022-05-18 上午3 01 57" src="https://user-images.githubusercontent.com/165936/168890347-94258977-3ef1-4b63-94b2-6b8638c5cac7.png">

**Desktop or Mobile (please complete the following information):**
 - OS: Mac
 - Browser Safari / Firefox
 - Version 

 **Additional Hardware (were you using headphones, an external speaker or an external microphone?):**
 - Type:
 - Model:

**Additional context**

**Comment by irvin** [comment]
found that the list of accents is not included in XHR result of https://commonvoice.mozilla.org/api/v1/language/accents when open this page

**Comment by mozgzh** [comment]
Thanks for filling the bug report @irvin - I checked the PR linked above. The list that accents was added to is actually deprecated, and the platform does not pull accent information from it anymore. Accents are now stored at the database level (in a table called `accents`) because we support both preset and user created accents. 

Suffice to say, a migration would need to be created to add present accents. The [migration would look similar to this one](https://github.com/common-voice/common-voice/blob/fef16a92ba054675354b35f9a1ea9ed34ccdc9c6/server/src/lib/model/db/migrations/20211110104012-accent-tables-in-db.ts), which adds data to the table (it also creates the table, but that does not need to happen again, just the inset login starting at line 274). 

Please feel free to submit a PR. Let me know if you have any questions or need assistance. Thanks!

**Comment by Heyhillary** [comment]
Hey @irvin,

Just following up on the convo - would you be interested in submitting a pr to resolve the issue please ?

If you need any support please let us know.

I look forward to hearing from you

**Comment by irvin** [comment]
@Heyhillary yes I still plan to make a PR for this, we still need this default set of data in order to make it more useful for researchers.

**Comment by irvin** [comment]
Create the PR to add the list into db table 
https://github.com/common-voice/common-voice/pull/4075

**Comment by irvin** [comment]
fixed by #4075
